### PR TITLE
Add the Output Argument

### PR DESCRIPTION
**Pull request by k1m0ch1**
Hi Love your tools, this might be useless but really helpful for the batch process

so the long story short I made this card for my kid, and then the teacher asks me if I can make some for the kids, well its generated anyway and very helpful, and the next day he asks me to make for a whole class, and next day he wants me to make for a whole school, and a weeks later other schools want me to help to make for a whole school.

more than 1000 generated file, with a custom filename for every kid and sent to the email
I'm doing it for free, while you made this free, love it <3
but the hard part is the automated batch

for the very first time it really easy to use the batch
- generate the worksheet
- rename the `worksheet.pdf` to some kid name from CSV file
and it will loop, and I found it annoying, so I made a new argument `--output` so I can generate the file without rename it.

image

### CODE REVIEW

**Comment by januschung** [comment]
Hey @k1m0ch1 thanks for using it and help me to help more people. This is what this is project is about. I will review the PR this weekend.

File: README.md [hunk 1]
```diff
@@ -43,6 +43,11 @@ python3 run.py --digits 3
 
 2. Print out the generated file `worksheet.pdf`
 
+For custom filename generated file, use the following command:
+```
+python3 run.py --output agnes-15oc.pdf
```
Comment (januschung) [inline README.md:48]: can you make it more generic please?

**Comment by januschung** [comment]
Please resolve conflicts. Otherwise lgtm.

File: README.md (hunk 1 above)
Comment (k1m0ch1) [reply README.md:48]: @januschung how about `quiz.pdf` ?

**Comment by k1m0ch1** [comment]
> Please resolve conflicts. Otherwise lgtm.

solved.

File: README.md (hunk 1 above)
Comment (januschung) [reply README.md:52]: Let's make it custom-name.pdf

Comment (k1m0ch1) [reply README.md:52]: sure

**Comment by k1m0ch1** [comment]
@januschung solved the filename to custom, please check again

**Review by januschung — APPROVED** [review]
Work like a charm. Thanks for making it better!

**Pull request merged**